domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ioc/InitiatePaymentResponse.java

```diff
@@ -1,12 +1,16 @@
 package uk.gov.hmcts.cmc.domain.models.ioc;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 @Builder
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Just FMI, do we need NoArgsConstructor here? I see that it's used in a test but don't understand purpose.

Reply:
You need a default constructor because jackson needs it. @Data and @Value already have some of the needed ones, when you see people being more explicit it's usually because of lombok + jackson issues, like https://github.com/rzwitserloot/lombok/issues/1563. Just don't add extra annotations and if it fails, you'll see why you need one.

Reply:
Thanks for explaining @silvia